src/main.rs

```diff
@@ -29,7 +29,7 @@ struct Cli {
     #[arg(short, long)]
     eval: Option<String>,
 
-    /// Generate docs
+    /// Amber-script [output folder (optional)]
```

Comment:
Amber-script? This flag generates docs. How about:

```suggestion
    /// Generate docs [OUTPUT DIR (default: `docs/`)]
```

Reply:
But the first parameter is the script file where will generate the documentation